tests/unit/inference/test_inference.py

```diff
@@ -415,6 +447,57 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+@pytest.mark.seq_inference_
```

Comment:
I think there's a typo here so that these tests are run in the inference workflow:
`@pytest.mark.seq_inference_` -> `@pytest.mark.seq_inference`